docs/openapi/components/schemas/common/FSMACreatingCTE.yml

```diff
@@ -0,0 +1,125 @@
+$linkedData:
+  term: FSMACreatingCTE
+  '@id': https://w3id.org/traceability#FSMACreatingCTE
+title: FSMA Creating CTE
+description: >-
+  "Information regarding an FSMA Creating CTE (Critical Tracking Event), designed to meet reporting
+  requirements for the FDA Traceability Rule as specified here:
+  https://www.fda.gov/food/food-safety-modernization-act-fsma/fsma-proposed-rule-food-traceability"
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - FSMACreatingCTE
+      - type: string
+        const: FSMACreatingCTE
+  foodCreated:
+    title: Food Created
+    description: The traceability food that was created.
+    $ref: ./FSMAProduct.yml
+    $linkedData:
+      term: foodCreated
+      '@id': https://w3id.org/traceability#FSMAProduct
+  locationCreated:
+    title: Location Created
+    description: Where the product was created.
+    $ref: ./Place.yml
+    $linkedData:
+      term: locationCreated
+      '@id': https://schema.org/Place
+  dateCompleted:
+    title: Date Completed
+    description: The date creation was completed in 24hr ISO 8601 format (e.g. 1970-01-01T00:00Z for UTC, or 1970:01:01T00:00-5:00 for New York on standard time).
+    type: string
+    $linkedData:
+      term: dateCompleted
+      '@id': https://schema.org/DateTime
```

Comment:
id type confusion.

Reply:
Find a `property` from this list: 

https://schema.org/Date

for example:

https://schema.org/dateCreated
https://schema.org/productionDate